gradle/native-build.gradle

```diff
@@ -13,7 +13,7 @@ def sdkPath(String platform) {
     return "/Applications/Xcode.app/Contents/Developer/Platforms/${platform}.platform/Developer/SDKs/${platform}${IOS_VERSION}.sdk";
 }
 
-def AndroidPlatform = 30
+def AndroidPlatform = 33
```

Comment:
We don't support this yet. From 31+ there are some changes that will be needed, so we can't jump just yet